pkgs/by-name/ol/ollama/package.nix

```diff
@@ -155,6 +156,7 @@ goBuild {
     [
       cmake
       gitMinimal
+      autoPatchelfHook
```

Comment:
Is this really needed?

If yes, then please let's move it to the end with a conditional:

```
++ lib.optionals (!stdenv.hostPlatform.isDarwin) autoPatchelfHook;
```